misago/users/utils.py

```diff
@@ -1,5 +1,15 @@
+from django.utils import timezone
+
+from misago.users.models.deleteduser import DeletedUser
+
 import hashlib
```

Comment:
Standard library imports should be on top of imports list. Please move your custom imports below `import hashlib`.